src/profile/v100/address/accesscontroller_address.rs

```diff
@@ -0,0 +1,217 @@
+use crate::prelude::*;
+
+use radix_engine_toolkit::models::canonical_address_types::CanonicalAccessControllerAddress as RetAccessControllerAddress;
+
+/// Address to an AccessController that controls an Account or Identity (Persona),
+/// it said entity has been "securified", e.g.:
+/// `"accesscontroller_rdx1c0duj4lq0dc3cpl8qd420fpn5eckh8ljeysvjm894lyl5ja5yq6y5a"`
+///
+/// When a user applies a SecurityStructureConfiguration for the first time on a
+/// non-securified entity (and signs and submit the resulting TX) said entity is
+/// "assigned" an AccessControllerAddress by the network.
+///
+/// An `AccessControllerAddress` has the [Scrypto's `EntityType`][entt] `GlobalAccessController`.
+///
+/// Implementation wise we wrap [Radix Engine Toolkit's `CanonicalAccessControllerAddress`][ret], and
+/// give it UniFFI support, as a `uniffi::Record` (we also own Serde).
+///
+/// [entt]: https://github.com/radixdlt/radixdlt-scrypto/blob/fc196e21aacc19c0a3dbb13f3cd313dccf4327ca/radix-engine-common/src/types/entity_type.rs
+/// [ret]: https://github.com/radixdlt/radix-engine-toolkit/blob/34fcc3d5953f4fe131d63d4ee2c41259a087e7a5/crates/radix-engine-toolkit/src/models/canonical_address_types.rs#L247-L248
+#[derive(
+    Clone,
+    Debug,
+    PartialEq,
+    Eq,
+    Hash,
+    derive_more::FromStr,
+    derive_more::Display,
+    SerializeDisplay,
+    DeserializeFromStr,
+    uniffi::Record,
+)]
+#[display("{secret_magic}")]
+pub struct AccessControllerAddress {
```

Comment:
Declare "manually" and NOT by macro so we can add documentation to it! which is not possible with `macro_rules!` (or at least too hard for me...).